moncli/entities/client.py

```diff
@@ -1356,6 +1358,36 @@ def add_users_to_workspace(self, workspace_id: str, user_ids: list, kind: Worksp
 
         return [en.User(creds=self.__creds, **data) for data in workspace_data]
 
+    def delete_users_from_workspace(self, workspace_id: str, user_ids: list(), *args: tuple, **kwargs: dict ):
+        """
+                Allows you to delete users to a workspace.
+
+                    Parameters 
+
+                        workspace_id: int
+                            The workspace's unique identifier
+                        user_ids: `list[str]`
+                                User IDs to subscribe to the workspace
+                        *args: tuple
+                            The collection of workspace return fields.
+                    
+                    Return Fields
+                
+                        id: Int	
```

Comment:
Please change these return fields to those of the User object.  You may copy and paste them from MondayClient.get_users().